src/ansible_navigator/image_manager/puller.py

```diff
@@ -128,8 +131,14 @@ def prologue_stdout(self):
         """print a little value added information"""
         messages = [("Execution environment image name:", self._image)]
         messages.append(("Execution environment image tag:", self._image_tag))
-        messages.append(("Execution environment pull policy:", self._pull_policy))
+        if isinstance(self._arguments, list):
+            arguments = " ".join(shlex.quote(arg) for arg in self._arguments)
```

Comment:
`shlex.join()`

Reply:
it's 3.8+ only, but I'll move this to utils so it's obvious.